Linux/Car_Client/chronoscomm.cpp

```diff
@@ -218,7 +220,7 @@ void ChronosComm::sendTraj(chronos_traj traj)
     for (chronos_traj_pt pt: points) {
         vb.vbAppendUint16(ISO_VALUE_ID_REL_TIME);
         vb.vbAppendUint16(4);
-        vb.vbAppendUint32(pt.tRel); // TODO: Multiply with 4?
+        vb.vbAppendUint32(pt.tRel.count()); // TODO: Multiply with 4?
```

Comment:
```suggestion
        vb.vbAppendUint32(pt.tRel.count()); 
```

This TODO was added by Vedder four years ago. I think it is safe to say we will not multiply by 4.